repos/iiif-storage/src/Repository/CollectionRepository.php

```diff
@@ -59,6 +84,23 @@ public function getDefaultQuery()
         return $query;
     }
 
+    public function getManifestMapFromCollection(int $collectionOmekaId)
+    {
+        $query = self::SELECT_MANIFESTS;
+        $statement = $this->connection->prepare($query);
+        $statement->bindValue('hasManifestsId', (int) $this->saturator->loadPropertyId('sc:hasManifests'));
+        $statement->bindValue('resourceId', (int) $collectionOmekaId);
+        $statement->execute();
+
+        $indexMap = [];
+        foreach ($statement->fetchAll() as $manifest) {
+            if ($manifest['uri']) {
+                $indexMap[$manifest['uri']] = $manifest['omeka_id'];
+            }
+        }
+        return $indexMap;
+    }
```

Comment:
This was written but not used in the end. The SQL up at the top is the quickest way to get a list of manifests on a collection, with omeka ids and labels.